root/components/home/Explore.js

```diff
@@ -0,0 +1,172 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+const Explore = (): React.Element<'section'> => (
+  <section className="bs" id="explore">
+    <div className="container pt-4 pb-4">
+      <div className="text-center pt-4">
+        <h2 className="mb-2 pb-2 text-uppercase fs-1 fw-bold">
+          {l('Explore MusicBrainz')}
```

Comment:
I was convinced this would be better in smallcaps, so that MusicBrainz is capitalized nicely, but I tried and I'm not sure it's an improvement:
![Screenshot from 2022-04-04 13-15-31](https://user-images.githubusercontent.com/1069224/161523821-41d4ecc3-0aae-4156-871a-a5c9fa825116.png)

Reply:
I am fine with keeping it all caps because then we can get away with keeping Music in purple color and Brainz in orange for this section because just above this we have done it in the About section.